plugins/kontent-plugin/scripts/setup.js

```diff
@@ -0,0 +1,99 @@
+const chalk = require("chalk")
+const argv = require("yargs-parser")(process.argv.slice(2))
+const fs = require("fs")
+const path = require("path")
+const inquirer = require("inquirer")
+const { ImportService, ZipService } = require("@kentico/kontent-backup-manager")
+const {
+  FileService,
+} = require("@kentico/kontent-backup-manager/dist/cjs/lib/node")
+
+const importData = async (projectId, managementApiKey) => {
+  const fileService = new FileService({
+    enableLog: true,
+  })
+
+  const zipFile = await fileService.loadFileAsync("data")
+
+  const zipService = new ZipService({
+    context: "node.js",
+    enableLog: true,
+  })
+
+  const importService = new ImportService({
+    onImport: (item) => {
+      console.log(`Imported: ${item.title} | ${item.type}`)
+    },
+    canImport: {
+      asset: (item) => true,
+      contentType: (item) => true,
+      assetFolder: (item) => true,
+      contentItem: (item) => true,
+      contentTypeSnippet: (item) => true,
+      language: (item) => true,
+      languageVariant: (item) => true,
+      taxonomy: (item) => true,
+    },
+    enablePublish: true,
+    projectId: projectId,
+    apiKey: managementApiKey,
+    enableLog: true,
+    fixLanguages: true,
+  })
+
+  const importData = await zipService.extractZipAsync(zipFile)
+
+  await importService.importFromSourceAsync(importData)
+}
+
+console.log(`
+  To use this starter, please create a new project in app.kentico.ai
+  and provide us with credentials need to set it up.
+  The required keys can be found in Project settings -> API KEYS and
+  you will need:
+
+    1. ${chalk.blue("Project ID")}
+    2. ${chalk.blue(
+      "API Management key"
+    )} needed for importing the content into the project
+
+  Everything ready? Let's do it!
+`)
+
+const questions = [
+  {
+    name: "projectId",
+    message: "Project ID",
+    when: !argv.projectId && !process.env.KONTENT_PROJECT_ID,
+  },
+  {
+    name: "managementApiKey",
+    when: !argv.managementApiKey,
+    message: "Your Content Management API access token",
+  },
+]
+
+inquirer
+  .prompt(questions)
+  .then(async ({ projectId, managementApiKey }) => {
+    // write env vars to .env.development & .env.production
+    const dotenv = [
+      `# All environment variables will be sourced`,
+      `# and made available to gatsby-config.js, gatsby-node.js, etc.`,
+      `# Do NOT commit this file to source control`,
+      `KONTENT_PROJECT_ID="${projectId}"`,
+    ].join("\n")
+
+    await importData(projectId, managementApiKey)
```

Comment:
A bit of a nit, but would be great to break this out into it's own `.then` block in the promise chain like we have setup in the [Contentful plugin](https://github.com/gatsbyjs/homepage-starters/blob/8845eab733ca7d79f8f570a091118ef563b8a4d5/plugins/contentful-plugin/scripts/setup.js#L113) `setup` script so that `.env` files get written with the values provided whether or not the import succeeds.